site/src/css/custom.css

```diff
@@ -62,6 +63,23 @@
   border-bottom: 2px solid var(--uitk-navigable-indicator-hover);
 }
 
+/* Override Salt accordion styles */
+
+.uitkAccordionSummary .uitkAccordionSummary-icon {
+  left: unset;
+  right: 0;
+}
+
+.uitkAccordionSummary .uitkAccordionSummary-icon:hover {
+  background: var(--accordion-summary-background-hover);
+}
+
+.uitkAccordionDetails .uitkAccordionDetails-content {
+  padding-left: 0;
+}
```

Comment:
Would it not make more sense to move these rules into `Accordion.module.css`?

Feels a bit risky to globally override them. What happens if we ever want to use a "vanilla" Salt accordion elsewhere on the site?

Reply:
The issue with moving these across to `Accordion.module.css` is that we can't access the icon class for example as it's not defined within that module

Reply:
This is using CSS Modules, right?

In that case, I think you can [use `:global(...)` to use "global" class names](https://github.com/css-modules/css-modules#exceptions) which CSS modules won't mangle then.

E.g.

```css
/* Inside Accordion.module.css */

.summary :global(.uitkAccordionSummary-icon) {
  /* ... */
}
```

Reply:
Ohh that works :) thank you!